ProteoWPFSuite/LoadResults.xaml.cs

```diff
@@ -86,6 +89,40 @@ public bool CK_rbneucode
                 populate_file_lists();
             }
         }
+        public bool CK_rbpromex
+        {
+            get
+            {
+                return ck_rbpromex;
+            }
+            set
+            {
+                if (ck_rbpromex == value)
+                    return;
+                ck_rbpromex = value;
+                rb_promex.IsChecked = (bool)ck_rbpromex;
+                PropertyChanged?.Invoke(this, new PropertyChangedEventArgs("CK_rbpromex"));
+                Sweet.lollipop.promex_deconv = (bool)ck_rbpromex;
+                populate_file_lists();
+            }
+        }
+        public bool CK_rbflashdeconv
```

Comment:
set this as default - right now when you click deconvolute neither CK is checked